book/intro.md

```diff
@@ -235,7 +235,7 @@ What makes that all work is the web browser's implementations of [inversion of
 control][inversion], [constraint programming][constraints], and
 [declarative programming][declarative]. The web _inverts control_, with an
 intermediary---the browser---handling most of the rendering, and the web
-developer specifying rendering parameters and content to this intermediary.
+developer specifying rendering parameters and contents to this intermediary.
```

Comment:
This sounds better to my ear.

Reply:
I think "content" is correct/better. It's how it's described in standards & developer circles.